app/views/govuk_component/search.raw.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  appearance_class ||= "on-light-background"
```

Comment:
This exposes the internal implementation to applications that are using it, so if we wanted to rename this class we couldn't easily.

We could have a 'inverse' flag or something else and then flip the class internally.